go/ct/spc/specification.go

```diff
@@ -1455,6 +1455,89 @@ func getAllRules() []Rule {
 		},
 	})...)
 
+	// --- CREATE2 ---
+
+	rules = append(rules, rulesFor(instruction{
+		op:        CREATE2,
+		name:      "_staticcall",
+		staticGas: 32000,
+		pops:      3,
+		pushes:    1,
+		conditions: []Condition{
+			Eq(ReadOnly(), true),
+		},
+		parameters: []Parameter{
+			ValueParameter{},
+			MemoryOffsetParameter{},
+			MemorySizeParameter{},
+			NumericParameter{},
+		},
+		effect: FailEffect().Apply,
+	})...)
+
+	rules = append(rules, rulesFor(instruction{
+		op:        CREATE2,
+		staticGas: 32000,
+		pops:      4,
+		pushes:    1,
+		conditions: []Condition{
+			Eq(ReadOnly(), false),
+		},
+		parameters: []Parameter{
+			ValueParameter{},
+			MemoryOffsetParameter{},
+			MemorySizeParameter{},
+			NumericParameter{},
+		},
+		effect: func(s *st.State) {
```

Comment:
`CREATE` and `CREATE2` effects are quite similar, do you think you could unify them ?